.github/README.md

```diff
@@ -6,5 +6,89 @@ This module was inspired by Azerothshard's Smartstone module.
+Unlocks `(Costume) Edwin VanCleef`
+
+> [!NOTE]
+> Some of the existing pets as example `Summon Hyjal Wisp`, will not work if you haven't completed their required raids in mythic from [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty).
+
+## How to make your own?
+
+### Pet
+
+Requires `creature_template`, `creature_template_model` and `smartstone_pets`.
+
+Example for `Amani'shi Groundbreaker` [here](https://github.com/chromiecraft/mod-chromiecraft-smartstone/blob/master/data/sql/db-world/updates/smartstone.sql#L108).
+
+### Combat Pet
+
+Requires `creature_template`, `creature_template_model`, `creature_template_spell` and `smartstone_pets`.
+
+Example for `Winterveil Helper` [here](https://github.com/chromiecraft/mod-chromiecraft-smartstone/blob/master/data/sql/db-world/updates/smartstone.sql#L326).
+
+### Costumes
+
+Requires only `smartstone_costumes`.
+
+Example for `Edwin VanCleef` [here](https://github.com/chromiecraft/mod-chromiecraft-smartstone/blob/master/data/sql/db-world/updates/smartstone.sql#L56).
+
+## Documentation
+
+### smartstone_pets (acore_world)
+
+| CreatureId     | Category | SubscriptionLevel | Duration          | Description | Enabled              |
+| -------------- | -------- | ----------------- | ----------------- | ----------- | -------------------- |
+| 80001 to 99999 | 0 or 1   | 1 to 3            | In Unix Timestamp | Gossip Text | 1 (Show) or 0 (Hide) |
+
+### smartstone_costumes (acore_world)
+
+| Id             | [DisplayId](https://www.azerothcore.org/wiki/creature_template_model#creaturedisplayid) | Category | SubscriptionLevel | Duration   | Description | Enabled              |
+| -------------- | --------------------------------------------------------------------------------------- | -------- | ----------------- | ---------- | ----------- | -------------------- |
+| 20001 to 29999 | ModelID                                                                                 | 1        | 1 to 3            | In seconds | Gossip Text | 1 (Show) or 0 (Hide) |
+
+### smartstone_categories (acore_world)
+
+| Id                | CategoryType | Title                        | SubscriptionLevel | [NPCTextId](https://www.azerothcore.org/wiki/npc_text#id) | Enabled              |
+| ----------------- | ------------ | ---------------------------- | ----------------- | --------------------------------------------------------- | -------------------- |
+| Unique Identifier | 0            | Title that shows a sub-menu? | 1 to 3            | npc_text.id                                               | 1 (Show) or 0 (Hide) |
+
+### smartstone_char_temp_services (acore_Characters)
+
+| PlayerGUID | ServiceId | Category | ActivationTime | ExpirationTime |
+| ---------: | --------: | -------: | -------------: | -------------: |
+|            |           |          |                |                |
+
+### character_settings (acore_Characters) (an example how a character without anything unlock would look)
```

Comment:
I'd say this table isnt normally human readable so its irrelevant to have it here (it doesnt add to anything)

Reply:
Which one smartstone_char_temp_services or character_settings (acore_Characters) or even both?

Reply:
Shall i restore `smartstone_char_temp_services` ?
i didnt knew how to fill it